src/main/java/net/mcreator/ui/action/AcceleratorDialog.java

```diff
@@ -44,6 +45,17 @@ static void showAcceleratorMapDialog(Window parent, AcceleratorMap acceleratorMa
 		map.setGridColor((Color) UIManager.get("MCreatorLAF.LIGHT_ACCENT"));
 		map.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
 
+		map.setDefaultRenderer(Object.class, new DefaultTableCellRenderer() {

```

Comment:
This should be wrapped with OS check so this does not run on Windows/-nix

Reply:
It may also be good enough to just set the font of the whole JTable as it is less complex